cmd/brew-test-bot.rb

```diff
@@ -977,7 +980,10 @@ def test_ci_upload(tap)
 
     if (pr = ENV["UPSTREAM_PULL_REQUEST"])
       pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-      safe_system "brew", "pull", "--clean", pull_pr
+      args = []
+      args << "--tap=#{@tap}" unless @tap.nil?
+      args << pull_pr
+      test "brew", "pull", "--clean", *args
```

Comment:
The custom argument option is to support non-GitHub or non-`homebrew-`-prefixed taps. In this case it is inferred but the result is not what you wish.



> Would you suggest adding a --tap option to the brew install --build-bottle command that is run by brew test-bot?

I would suggest you add it your e.g. CircleCI configuration.